### DISCUSSION TITLE
Easiest way to turn a single-speaker model into a multi-speaker model?

### DISCUSSION THREAD

**Discussion by fijipants** [coqui-ai/TTS#1425] [General Q&A] [answered]
I've trained a single-speaker model, and I'm able to finetune it on other single-speaker data using `--continue_path`.

Now I want to finetune it on multi-speaker data. If my understanding is correct, `--continue_path` won't work properly because the speaker embedding layer is missing(?)

Is it possible to convert it to a multi-speaker model and continue from it?

**Comment by fijipants** [comment, marked as answer]
It turns out using `--continue_path` works properly, who'd have thought.

**Reply by arif334** [reply]
Interesting!
